plugins/crd/options.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package netmeshplugincrd
+
+import (
+	"reflect"
+	"sync"
+
+	"github.com/ligato/networkservicemesh/plugins/objectstore"
+
+	"github.com/ligato/networkservicemesh/plugins/handler"
+
+	"github.com/ligato/networkservicemesh/plugins/logger"
+	"github.com/ligato/networkservicemesh/utils/command"
+)
+
+const (
+	// DefaultName of the netmeshplugincrd.Plugin
+	DefaultName = "netmeshplugincrd"
+	// KubeConfigFlagName - Cmd line flag for specifying kubeconfig filename
+	KubeConfigFlagName = "kube-config"
+	// KubeConfigFlagDefault - default value of KubeConfig
+	KubeConfigFlagDefault = "/etc/kubernetes/admin.conf"
```

Comment:
Since netmesh will mostly run `in-cluster` I am not sure why defaulting to `/etc/kubernetes/admin.conf` The default should be `""` and start up parameter should be used to provide location of kubeconfig if netmesh runs out of cluster mode.

Reply:
@sbezverk Happy to make that change :)

Reply:
@sbezverk Done.